packages/jni-swig-stub/realm.i

```diff
@@ -153,6 +153,10 @@ struct realm_size_t {
 // bool output parameter
 %apply bool* OUTPUT { bool* out_found };
 
+// uint64_t output parameter
+// FIXME How to convert uint64_t to Long or ULong?
```

Comment:
@rorbech I could use some help with figuring this out...the SWIG syntax is a bit magical to me 🙈

Reply:
You can tell Swig to copy a typemap from an existing type to another by ex:
```
%apply int64_t* OUTPUT { uint64_t* out_long };
```
Which will copy the typemap from an "output int64_t pointer" to all `unint64_t*` variables named `out_long`. 

`ULong` is an experimental add-on in Kotlin, so we cannot use that in JNI code.